config/notify.config.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+/**
+ * Config values used for GOV.UK Notify
+ * @module NotifyConfig
+ */
+
+// We require dotenv directly in each config file to support unit tests that depend on this subset of config.
+// Requiring dotenv in multiple places has no effect on the app when running for real.
+require('dotenv').config()
+
+const config = {
+  apiKey: process.env.GOV_UK_NOTIFY_API_KEY || 'test-key'
```

Comment:
IMHO, if a config value is required, we should leave off the default.

I'm guessing this will break your tests, certainly in CI. In which case, you can set a fake API key in `.github/workflows/ci.yml`.

Also, we need to add `GOV_UK_NOTIFY_API_KEY` to `.env.example`.